vxquery-core/src/main/java/org/apache/vxquery/xmlquery/translator/XMLQueryTranslator.java

```diff
@@ -1969,6 +1970,13 @@ private ILogicalExpression normalize(ILogicalExpression expr, SequenceType type)
         }
     }
 
+    private ILogicalExpression data(ILogicalExpression expr) throws SystemException {
+        ILogicalExpression atomizedExpr = new ScalarFunctionCallExpression(BuiltinFunctions.FN_DATA_1,
```

Comment:
Since this function is one statement, remove the variable and make that the return statement.